kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -175,6 +175,14 @@
         <AD id="container.restart.onfailure" name="Restart Container On Failure" type="Boolean" cardinality="1" required="true" default="false"
             description="Automatically restart the container when it has failed.">
         </AD>
+        
+        <AD
+            id="enforcement.digest"
+            name="Container Image Enforcement Digest "
+            description="Digest of the image wanted to be run, added to the Enforcement Allowlist of the Container Orchestration Service, if the enforcement monitoring is enabled. If not given, it will be computed by the Signature Verification of the service."
```

Comment:
I find this statement somewhat confusing... 

> Digest of the image wanted to be run, added to the Enforcement Allowlist of the Container Orchestration Service, if the enforcement monitoring is enabled

Does it mean that I have to put the digest in the Container Orchestration Service allowlist by myself? (I know it doesn't). Does the user need to know about the enforcement allowlist in this instance? I would propose something like:

```suggestion
            description="Digest of the container image allowed to run on the device if the Container Enforcement Monitor is enabled. If not given, it will be computed by the Container Signature Verification service."
```

Reply:
Yeah I agree that the user is not required to know which is the underlying process. I commit the change